packages/design-system/src/components/FieldContainer/FieldContainer.jsx

```diff
@@ -54,10 +54,6 @@ export class FieldContainer extends React.PureComponent {
       setRef: this.setFieldRef,
     };
 
-    // Render children with render props or normally
-    // We allow normal rendering to simplify unit tests in ChoiceField
-    const render = typeof children === 'function' ? children(fieldInputProps) : children;
```

Comment:
Originally I allowed both normal ReactNode and functions for the `children` to avoid breaking tests in `ChoiceList` and `MonthPicker`, but only allowing function is a better solution overall